Userland/Libraries/LibGUI/Action.h

```diff
@@ -76,6 +78,7 @@ class Action final : public Core::Object {
     void set_status_tip(String status_tip) { m_status_tip = move(status_tip); }
 
     Shortcut shortcut() const { return m_shortcut; }
+    Shortcut alternate_shortcut() const { return m_alternate_shortcut; }
```

Comment:
This (and also `shortcut()`) could be returned with as `const&` here. It's not a big deal, since the `Shortcut` class is quite small, though.

Reply:
Great suggestions. I added both of them. Took a bit of learning git rebasing. Hopefully everything came out alright.